tasks/monitoring/systemd.yml

```
@@ -0,0 +1,71 @@
+---
+- name: Create monitoring user
+  user:
+    name: bbb-exporter
+    shell: /usr/sbin/nologin
+    home: "{{ bbb_monitoring_systemd_directory }}"
+    create_home: false
+
+- name: Deploy git repository to host
+  git:
+    repo: https://github.com/greenstatic/bigbluebutton-exporter.git
+    dest: "{{ bbb_monitoring_systemd_directory }}"
+    version: "{{ bbb_monitoring_systemd_version }}"
+
+- name: Change ownership directory
+  file:
+    path: "{{ bbb_monitoring_systemd_directory }}"
+    state: directory
+    owner: bbb-exporter
+    group: bbb-exporter
+    more: '0755'
```

Typo, is `more` should be `mode` :D
Thanks @EdJoPaTo for the github actions ❤️

---

For even more (hihi) fancy tooling, there are the suggestions that can be accepted (as a bulk). Especially helpful for these kinds of small suggestions.
```suggestion
    mode: '0755'
```

Not sure why this one wants to replace a free line instead of the correct `more` line 🧐